site/en/docs/extensions/mv3/getstarted/extensions-101/index.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Extensions 101'
+subhead: 'Explore basic concepts of Chrome extension development.'
+description: 'Explore basic concepts of Chrome extension development.'
+date: 2022-08-10
+# updated: 2022-06-13
+---
+
+## Overview {: #overview }
+
+Chrome extensions enhance the browsing experience by adding features and functionality to the Chrome
+browser. This page introduces Chrome extension development and provides links to step-by-step
+[beginner tutorials][section-tutorials]. 
+
+## Web technologies {: #web-tech }
+
+Extensions are written with the same web technologies used to create web applications:
+
+- [HTML][mdn-html] is used as a content markup language.
+- [CSS][web-dev-css] is used for styling.
+- [JavaScript][mdn-js] is used for scripting and logic.
+- Extensions can use all the [JavaScript APIs](https://developer.mozilla.org/docs/Web/API) that the
+  browser provides.
+
+<!-- Should we use https://developer.chrome.com/docs/extensions/api_other/ ?-->
```

Comment:
I don't think so. Actually, I'd suggest we remove that page from the MV3 doc set as I'm worried it's emphasis on DOM capabilities may do more harm than good.

Reply:
Following up on a conversation from earlier in the week; while we could refactor and update this page, I think that would be too much to try to tackle as part of this update.

As I recall, @AmySteam mentioned that this page is referenced in ~5 other places. It may be best to defer a decision on what to do with that page until we have a chance to take a closer look at how it's being used, what alternatives exist, and whether it's worth the investment to revise this page.

Reply:
I opened [PR #3468](https://github.com/GoogleChrome/developer.chrome.com/pull/3468)

These are the only files that link to it, and the MDN doc will work for each case:

- [More APIs](https://deploy-preview-3468--developer-chrome-com.netlify.app/docs/extensions/reference/experimental#other) in Experimental APIs.
- [What tech are used](https://developer.chrome.com/docs/extensions/mv3/faq/#faq-gen-02) in Extensions FAQs
- [More information](https://developer.chrome.com/docs/extensions/mv2/devtools/#more) in MV2 Developing Devtools